src/components/NavbarLink.tsx

```diff
@@ -0,0 +1,19 @@
+import { NavLink } from 'react-router-dom';
+import { Pages } from '../types/Pages';
+import { getLinkClass, getLinkStyle } from './utils/Utils';
+
+type Props = {
+  page: string;
+};
+
+export const NavbarLink: React.FC<Props> = ({ page }) => {
+  return (
+    <NavLink
+      to={page === Pages.Home ? '/' : `/${page.toLowerCase()}`}
```

Comment:
this condition only applies to two links and does not scale
makes sense to create separate links to "Home" and "People"